doc/GettingStartedLinux.md

```diff
@@ -0,0 +1,81 @@
+# Getting Started on Linux
+
+1. Install Java and Maven
+
+  Go to [Java SE Downloads page](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+  and get the URL to download the RPM, e.g.:
+
+  ```sh
+  wget http://download.oracle.com/otn-pub/java/jdk/7u40-b43/jdk-7u40-linux-x64.rpm?AuthParam=11232426132 -o jdk-7u40-linux-x64.rpm
+  rpm -Uvh jdk-7u40-linux-x64.rpm
+  ```
+
+  Or install via `yum` or `apt-get`:
+
+  ```sh
+  sudo yum install java-devel
+  ```
+
+  Download and install [Maven](http://maven.apache.org/download.cgi), e.g.:
+
+  ```sh
+  wget http://ftp.heanet.ie/mirrors/www.apache.org/dist/maven/maven-3/3.1.1/binaries/apache-maven-3.1.1-bin.tar.gz
```

Comment:
We should include a note about verifying the download.

i.e. https://www.apache.org/info/verification.html